build.rs

```diff
@@ -91,6 +91,10 @@ fn build_zlib() {
 fn make() -> Command {
     let cmd = if cfg!(any(target_os = "freebsd", target_os = "dragonfly")) {"gmake"} else {"make"};
     let mut cmd = Command::new(cmd);
+    if let Some(makeflags) = env::var_os("CARGO_MAKEFLAGS") {
+        cmd.env("MAKEFLAGS", makeflags);
+    }
```

Comment:
Should this perhaps be combined with the `cfg!` below, as they look quite related? In that I'd imagine that the windows cfg would come first, and only if we're not on windows do we try to set `MAKEFLAGS`?

Reply:
Well It adds an indentation and if `cfg!(windows)` is true, setting the `MAKEFLAGS` is a noop since it's removed just after. I can put it in an else statement if you want though

Reply:
Er what I mean is --

```rust
// comment
if cfg!(windows) {
    cmd.env_remove(...);
} else if let Some(..) = .. {
    cmd.env(..., ...);
}
```

Reply:
Oh, right, it's better